### DISCUSSION TITLE
Link on Homepage does not work

### DISCUSSION THREAD

**Discussion by JulianKle** [neuefische/questions#798] [Web] [answered]
I have outsourced styling components in my app, including the link that can be found on my homepage under "New Assessment" (NewLink). Since the styling outsourcing, the link is no longer clickable as long as no assessment has been added. The conditional rendering in the return part cannot actually affect the link, but if you exclude it, the link works again. What could be the problem?

Link to my index.js: https://github.com/JulianKle/capstone-project/blob/main/pages/index.js

Many thanks in advance

**Comment by onemanwenttomow** [comment, marked as answer]
I think the problem is a CSS one. 

Currently the way you have things laid out, the section is appearing on top of the link. If you remove the z-index 1 from that section (i.e. `StyledContentWithoutAssessments`) the problem goes away.

**Reply by JulianKle** [reply]
Perfect, thank you!
